### PR TITLE
Closes #1348: Remove deprecated ParagraphsBehavior migrate process plugin.

### LINKED ISSUE #1348: Remove modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php
Remove modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php since it is deprecated in https://github.com/az-digital/az_quickstart/pull/1345/files

## Conditions of satisfaction
- [ ] modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php deleted.

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Hoping to deprecate this plugin in favor of the one here: https://github.com/az-digital/az_quickstart/pull/1345/files#diff-df90683b10b3f30addedcf7829163c29bf81d810abf564c20ae743018e3c825a

If deprecated in 2.1.3 we can remove in 2.2 or 2.3

## Related Issue
Close #1348 
Blocked by #1345 

## How to test:
Since we are removing a plugin, we need to make sure that all of our migrations still work.
Luckily we have automated tests for most or all of the migrations
There should not be any errors in probo during quickstart install
There should not be any errors in probo during the migration test
There should not be any errors in probo during phpunit test
Text area page should have all paragraph behavior settings match modules/custom/az_demo/data/az_demo_text_paragraph.json
Text with background page should have all paragraph behavior settings match modules/custom/az_demo/data/az_demo_text_background_paragraph.json

## Types of changes
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by joeparsons — DISMISSED** [review]

**Comment by joeparsons** [comment]
Should we remove it from the README as well?
https://github.com/az-digital/az_quickstart/tree/main/modules/custom/az_migration#migrate-plugins

**Comment by trackleft** [comment]
I kind of like how Drupal points out when a change occurred and how to migrate to that change (In regards to their deprecations)
image

**Comment by trackleft** [comment]
Ok I've changed the readme to be more helpful with migrating from `paragraphs_behavior_settings` to `az_paragraphs_behavior_settings`

**Review by joeparsons — APPROVED** [review]

**Comment by akslay** [comment]
Should there actually be content on these pages? If there's supposed to be content, I think something went awry for:

- Card Decks
- Accoordions
- Photo Galleries

https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1349.probo.build/pages/card-decks

**Comment by trackleft** [comment]
> Should there actually be content on these pages? If there's supposed to be content, I think something went awry for:
> 
> * Card Decks
> * Accoordions
> * Photo Galleries
> 
> [7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1349.probo.build/pages/card-decks](https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1349.probo.build/pages/card-decks)

We don't have content for those pages yet.

**Comment by akslay** [comment]
Is this also a pre-existing issue (unrelated to your PR)?
https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1349.probo.build/admin/reports/dblog/event/130

**Comment by trackleft** [comment]
Yes, that exists here: http://demo.quickstart.arizona.edu/pages/text as well

**Review by akslay — DISMISSED** [review]

File: modules/custom/az_migration/README.md [hunk 1]
```diff
@@ -529,7 +529,9 @@ These plugins are designed to be reusable in custom migrations.
 - [MigratedPathLookup (az_migrated_path_lookup)](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_migration/src/Plugin/migrate/process/MigratedPathLookup.php)
 - [TextFormatRecognizer (text_format_recognizer)](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_migration/src/Plugin/migrate/process/TextFormatRecognizer.php)
 - [ParagraphsMappingFlexiblePage (paragraphs_mapping_flexible_page)](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_migration/src/Plugin/migrate/process/ParagraphMappingFlexiblePage.php)
-- [ParagraphsBehavior (paragraphs_behavior_settings)](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php) (Deprecated: use `az_paragraphs_behavior_settings`)
+- [ParagraphsBehavior
+  (paragraphs_behavior_settings)](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php)
```
Comment (trackleft) [inline modules/custom/az_migration/README.md:533]:
```suggestion
  (paragraphs_behavior_settings)](https://github.com/az-digital/az_quickstart/blob/2.2.x/modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php)
```

Comment (trackleft) [reply modules/custom/az_migration/README.md:533]: Sorry

Comment (joeparsons) [reply modules/custom/az_migration/README.md:533]: Worth changing IMHO.

**Review by joeparsons — APPROVED** [review]

**Review by akslay — APPROVED** [review]

**Pull request merged**